internal/pkg/addon/package.go

```diff
@@ -193,9 +191,13 @@ var resourcePackageConfig = map[string][]packagePropertyConfig{
 }
 
 func (t *cfnTemplate) pkg(a *Addons) error {
-	resources := mappingNode(&t.Resources)
+	err := a.packageTransforms(t.Metadata, t.Parameters, t.Mappings, t.Conditions, t.Transform, t.Resources, t.Outputs)
```

Comment:
> We can use the AWS::Include in any section of the template, except Parameters. 
https://medium.com/swlh/using-the-cloudformation-aws-include-macro-9e3056cf75b0

This is such a funky transformation, I wonder how many people use it

Reply:
whats extra weird to me is how the `aws cloudformation package` doesn't package an `AWS::Include` if it's the top level transform 🤔 
like:
```
Transform:
  Name: AWS::Include
  Parameters:
    Location: ./tags
Resources:
# blah
Outputs:
# bleh
```
just leaves `Transform.Parameters.Location` as is :joy: